packages/cna-template/template/nuxt/nuxt.config.js

```diff
@@ -1,21 +1,16 @@
 <%_ if (ui === 'vuetify') { _%>import colors from 'vuetify/es5/util/colors'
-<%_ } _%>
 
-export default {
-  /*
-  ** Nuxt rendering mode
-  ** See https://nuxtjs.org/api/configuration-mode
-  */
-  mode: '<%= mode %>',
-  /*
-  ** Nuxt target
-  ** See https://nuxtjs.org/api/configuration-target
-  */
+<%_ } _%>export default {
+  <%_ if (mode === 'spa') { _%>
+  // Disable server-side rendering (https://go.nuxtjs.dev/config-mode)
```

Comment:
We need a new link

Reply:
Like https://go.nuxtjs.dev/config-ssr ?

Reply:
Yes! Debbie just created page i think (not merged)